packages/marketplace/contracts/ExchangeCore.sol

```diff
@@ -134,13 +140,13 @@ abstract contract ExchangeCore is Initializable, ITransferManager {
     function _validateOrders(
         address sender,
         LibOrder.Order memory orderLeft,
-        bytes memory signatureLeft,
+        Signature memory signatureLeft,
         LibOrder.Order memory orderRight,
-        bytes memory signatureRight
+        Signature memory signatureRight
     ) internal view {
         // validate must force order.maker != address(0)
-        orderValidator.validate(orderLeft, signatureLeft, sender);
-        orderValidator.validate(orderRight, signatureRight, sender);
+        orderValidator.validate(orderLeft, signatureLeft.signature, sender, signatureLeft.version);
+        orderValidator.validate(orderRight, signatureRight.signature, sender, signatureRight.version);
```

Comment:
you could pass the whole signatureRight struct intead of each field

Reply:
done -> https://github.com/thesandboxgame/sandbox-smart-contracts/pull/1385/commits/815766646c98bf962eec4dac998da27dfd78991f